doc/advance_install.rst

```diff
@@ -0,0 +1,267 @@
+  sudo useradd -m -s /bin/bash orga1-user
+  sudo useradd -m -s /bin/bash orga2-user
+
+pytition-admin sera l'utilisateur dédié à la maintenance du code de pytition
+
+.. code-block:: bash
+  sudo mkdir -p /etc/pytition/{orga1,orga2,admin}
+  sudo touch /etc/pytition/{orga1,orga2,admin}/__init__.py
+  sudo touch /etc/pytition/__init__.py
+
+Le repertoitre /etc/pytition/ contient la configuration de la base de donnée et la configuration de pytition pour chaque site
+
+.. code-block:: bash
+  sudo mkdir -p /srv/pytition/www/mediaroot/{admin,orga1,orga2}
+  sudo mkdir -p /srv/pytition/www/static
+
+
+Install system dependencies:
+============================
+
+.. code-block:: bash
+
+  $ sudo apt update
+  $ sudo apt install git virtualenv python3-dev build-essential default-libmysqlclient-dev gettext libzip-dev libssl-dev apache2 uwsgi
+
+Get the source, configure and initialize Pytition
+=================================================
+
+Get the latest release git tag:
+
+.. code-block:: bash
+
+  $ version=$(curl -s https://api.github.com/repos/pytition/pytition/releases/latest | grep "tag_name" | cut -d : -f2,3 | tr -d \" | tr -d ,)
+
+
+Create a Python3 virtualenv to install Pytitiont's dependencies:
+
+.. code-block:: bash
+
+  cd /srv/pytition/
+  sudo virtualenv -p python3 pytition_venv
+
+Clone Pytition git repository and checkout latest release:
+
+.. code-block:: bash
+
+  cd www
+  sudo git clone https://github.com/pytition/pytition
+  cd pytition
+  sudo git checkout $version
+
+Attribuer les bons propriétaires et les bons droits aux dossiers:
+.. code-block:: bash
+  sudo chown -R pytition-admin:www-data /srv/pytition
+  sudo chown orga1-user:www-data /srv/pytition/www/mediaroot/orga1
+  sudo chown orga2-user:www-data /srv/pytition/www/mediaroot/orga2
+  sudo chmod g+s /srv/pytition/www/static/
+
+Enter your virtualenv and install Pytition's dependencies:
+
+.. code-block:: bash
+  sudo su pytition-admin
+  source /srv/pytition/pytition_venv/bin/activate
+  (pytition_venv) $ pip3 install -r /srv/pytition/www/pytition/requirements.txt
+
+Créer les bases de données db-pytition-orga1, db-pytition-orga2, db-pytition-admin ainsi 
+que les utilisateurs associés db-user-orga1, db-user-orga2 et db-user-admin sur votre serveur MariaDB
+
+Pour chaque organisation, écrire le fichier /etc/pytition/{orga1,orga2,admin}/my.cnf
+Exemple de fichier pour orga1
+  [client]
+  host = your-data-base-server
+  database = db-pytition-orga1
+  user = db-user-orga1
+  password = YOUR_PASSWORD_HERE
+  default-character-set = utf8
+
+Pour l'admin, on pourra utiliser une base sqlite3 plutôt que de créer une nouvelle base sur les serveur mariaDB
+
+Pour chaque organisation, créer le fichier /etc/pytition/{orga1,orga2,admin}/config.py en copiant par exemple le fichier 
+/srv/pytition/www/pytition/config_example.py
+
+Les fichiers my.cnf et config.py doivent avoir les bonnes permissions et droits. Par exemple pour orga1:
```

Comment:
The `my.cnf` and `config.py` files must have the correct permissions. E.g. for orga1: